echo/wavefront-tracing-utils.go

```diff
@@ -0,0 +1,173 @@
+package echo
+
+import (
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	reporter "github.com/wavefronthq/wavefront-opentracing-sdk-go/reporter"
+	"github.com/wavefronthq/wavefront-opentracing-sdk-go/tracer"
+	wfTracer "github.com/wavefronthq/wavefront-opentracing-sdk-go/tracer"
+	wfApplication "github.com/wavefronthq/wavefront-sdk-go/application"
+	"github.com/wavefronthq/wavefront-sdk-go/senders"
+)
+
+// Tracer Global Tracer
+var Tracer opentracing.Tracer
+var globalTracerConfig *tracerConfig
+
+// InitTracer initialize tracer object, initialize the GlobalTracer with this newly created,
+// tracer object.
+// Call utils/wavefront.Tracer to access the tracer object
+//For proxy sender send token as empty string
+func InitTracer(configFilePath string, routesRegistrationFilePath string, echoWeb *echo.Echo, sendDirectly bool, server string, token string, flushIntervalSeconds int) error {
```

Comment:
Couple of comments:

- It'd be cleaner for this constructor to take a config struct rather than have so many args and not all of which may be needed.
- Second, we should just leverage the constructs defined within the wavefront-sdk-go when possible rather than introducing new ones. This will allow users to customize all of the properties (flush interval, batch size etc) as relevant.

Basically define something like:
```
type Config struct {
  DirectCfg *senders.DirectConfiguration
  ProxyCfg *senders.ProxyConfiguration
  CfgFile string
  RoutesFile string
  ...
}
```

Reply:
Done